test/system/test_tpm2_create.sh

```diff
@@ -47,6 +47,17 @@ cleanup() {
 }
 trap cleanup EXIT
 
+# Certain TPM 2.0 chip, e.g, Nationz Z32H320TC, and tpm2simulator referring
+# to spec 1.16 may have an errata, disallowing both decrypt and sign set for
+# a symcipher object, and in response to RC_ATTRIBUTES. In order to work
+# around it, we attempt to run tpm2_create again with sign bit clear.
+create() {
+    tpm2_create $@ 2>&1 |
+        grep -q 'Create Object Failed ! ErrorCode: 0x2c2' && {
+            tpm2_create -Q -A 0x20072 $@
```

Comment:
Why not use the friendly name attributes here?

Reply:
@williamcroberts It is unsupported to use friendly naming scheme for the attributes option. The option 'A' for tpm2_createprimary supports.